### DISCUSSION TITLE
Print limitation to 25 pages

### DISCUSSION THREAD

**Discussion by jrevillard** [outline/outline#5340] [Self Hosting] [answered]
Hi,

I see that if a document have more than 25 pages, it will not be printed entirely... is there a reason for that ? Could this limit be removed ?

Best,
Jerome

**Comment by tommoor** [comment, marked as answer]
I'm not aware of any such limitation – it could be your browser?

**Reply by jrevillard** [reply]
I don't think so but I will check more carefully. Could it come from https://github.com/outline/outline/blob/7ce97f4d50463478f25087e0aaf57808f7fa92f9/app/stores/BaseStore.ts#L23 ?

**Reply by jrevillard** [reply]
I confirm that this is not my browser.... I tried with another one and I have the same issue.

Do you think that I pointed to the good place in the source code @tommoor ?

Best,
Jerome

**Reply by tommoor** [reply]
It doesn't make any sense to me, that code is used for pages of API results – not pages of a document, documents are not paginated at all.

**Reply by jrevillard** [reply]
Forget about it.. today, I made another test and it worked adding plenty of text in a document. There is certainly something specific to the document that I tried to print which produced this behavior but I cannot make more tests for the moment. Thank you anyway !
